package/MDAnalysis/analysis/density.py

```diff
@@ -425,7 +425,7 @@ def density_from_trajectory(*args, **kwargs):
 
 def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
                           metadata=None, padding=2.0, cutoff=0, soluteselection=None,
-                          use_kdtree=True, **kwargs):
+                          use_kdtree=True,sel_update=None, **kwargs):
```

Comment:
Please change the name from `sel_update` to `update_selection` so that this is more consistent with what we have for `MDAnalysis.analysis.hbonds.HydrogenBondAnalysis`.